test/UI/UI_JobRolePage.js

```
@@ -0,0 +1,42 @@
+var axios = require('axios');
+var MockAdapter = require('axios-mock-adapter');
+var chai = require('chai');  
+const expect = chai.expect;
+
+const { Builder, By, Key, until } = require('selenium-webdriver')
+const assert = require('assert');
+const { threadId } = require('worker_threads');
+const { delay } = require('lodash');
+const { resolve } = require('path');
+const { WebDriver } = require('selenium-webdriver');
+
+describe('JobRolesCheck', function() {
+  this.timeout(30000)
+  let driver
+  let vars
+  beforeEach(async function() {
+    driver = await new Builder().forBrowser('chrome').build()
+    vars = {}
+  })
+  afterEach(async function() {
+    await driver.quit();
+  })
+  it('JobRolesCheck', async function() {
+
```

perhaps 'JobRolesCheck' isn't as descriptive as it could be relative to what is being tested here?

---

changed to 'should return an error when Job Roles page is not displayed or there is no Job Roles table'

---

change test name to the good case, ie what are you testing for, if this fails it will tell someone how to fix it. Imagine if you left the company